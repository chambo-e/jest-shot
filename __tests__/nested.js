import shot from '../src/';
import Nested from '../__components__/nested';

const mutations = [
	{
		description: 'without props, should snap shallow',
	},
	{
		description: 'without props, should snap full tree',
		mount: true,
	},
];

shot(Nested, mutations);
