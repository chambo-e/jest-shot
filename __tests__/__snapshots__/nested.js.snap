// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NestedComponent without props, should snap full tree 1`] = `
<NestedComponent>
  <div>
    <SimpleComponent>
      <div>
        Hello
      </div>
    </SimpleComponent>
    World
  </div>
</NestedComponent>
`;

exports[`NestedComponent without props, should snap shallow 1`] = `
<div>
  <SimpleComponent />
  World
</div>
`;
